wmn-data.json

```diff
@@ -4343,6 +4379,18 @@
         "cat" : "tech",
         "valid" : true
        },
+       {
+        "name" : "Peing",
+        "uri_check" : "https://peing.net/api/v2/items/?type=answered&account={account}",

```

Comment:
This is a great check_uri. I'm wondering if there is a link to the user's profile that we can add in the `pretty_uri` field to take users right to it instead of to this JSON

Reply:
The way that peing works is that if you were to request a user normally, using the URL [https://peing.net/honey](https://peing.net/honey), for example, would redirect you to a different site along the structure of https://peing.net/<locale>/honey, where locale is the "locale" token in the API request. In this case, we get redirected to [https://peing.net/ja/honey](https://peing.net/ja/honey). Using the API Directly here is the best option, as it does not have to do the gymnastics of figuring out what the account's locale is.